deploy/charts/istio-csr/README.md

```diff
@@ -267,7 +267,7 @@ Create the default certificate as part of install.
 > 2048
 > ```
 
-Number of bits to use for istiod-tls RSAKey
+Number of bits to use for istiod-tls Key
```

Comment:
comment: as mentioned in standup, this is the wrong abstraction for ECDSA but this kind of thing is used in istio and cert-manager so it's totally fine here!

Reply:
I lied a little bit and it turns out they do use: "ECC_CURVE" : "P384" or 256 currently in Istio.
However it is true that the only supported ecc curves will be those two for the foreseable future!
If you want I can add a new key for ecc_curve and change it, it's not biggie